### PR TITLE
Add Cryptoradar to exchange list

### PR DESCRIPTION
**Pull request by josiasmontag**
[Cryptoradar](https://cryptoradar.co) is a tool which compares the prices, fees and features of different exchanges. 
It could help users to filter this long list of exchanges based on their individual criteria.

### CODE REVIEW

File: _templates/exchanges.html [hunk 1]
```diff
@@ -422,3 +422,6 @@ <h3 id="south-africa" class="no_toc">South Africa</h3>
 <p class="introlink exchanges-introlink">Visit
   <a href="https://coinatmradar.com/">Coin ATM Radar</a> to find local Bitcoin ATMs.
 </p>
+<p class="introlink exchanges-introlink">Visit
+  <a href="https://cryptoradar.co">Cryptoradar</a> to compare above exchanges based on prices, fees and features.
```
Comment (wbnns) [inline _templates/exchanges.html]: Let's combine with with the first link to Buy Bitcoin Worldwide and write it like so:

image

**Comment by josiasmontag** [comment]
Sounds good, done!

File: _templates/exchanges.html [hunk 2]
```diff
@@ -417,11 +417,8 @@ <h3 id="south-africa" class="no_toc">South Africa</h3>
 
 
 <p class="introlink exchanges-introlink exchanges-buy-link">Visit
-  <a href="https://www.buybitcoinworldwide.com/">Buy Bitcoin Worldwide</a> for user reviews on some of the above exchanges.
+  <a href="https://www.buybitcoinworldwide.com/">Buy Bitcoin Worldwide</a> for user reviews on some of the above exchanges, or <a href="https://cryptoradar.co">Cryptoradar</a> for comparison based on prices, fees and features.
```
Comment (wbnns) [inline _templates/exchanges.html]: @josiasmontag Could you please make the word "comparison" pluralized?

Comment (josiasmontag) [reply _templates/exchanges.html]: Done

**Comment by wbnns** [comment]
@josiasmontag Thanks!

This is scheduled to be merged on Monday, July 16th.

**Pull request merged**